examples/examples_test.go

```diff
@@ -28,22 +48,45 @@ func getCwd(t *testing.T) string {
 }
 
 func getBaseOptions() integration.ProgramTestOptions {
-	return integration.ProgramTestOptions{}
-}
-
-func getJSBaseOptions(t *testing.T) integration.ProgramTestOptions {
-	base := getBaseOptions()
-	baseJS := base.With(integration.ProgramTestOptions{
+	return integration.ProgramTestOptions{
 		ExpectRefreshChanges: true,
 		Config: map[string]string{
 			"rabbitmq:endpoint": "http://127.0.0.1:15672",
 			"rabbitmq:username": "guest",
 			"rabbitmq:password": "guest",
 		},
+	}
+}
+
+func getJSBaseOptions() integration.ProgramTestOptions {
+	base := getBaseOptions()
+	baseJS := base.With(integration.ProgramTestOptions{
 		Dependencies: []string{
 			"@pulumi/rabbitmq",
 		},
 	})
 
 	return baseJS
 }
+
+func getPythonBaseOptions() integration.ProgramTestOptions {
+	base := getBaseOptions()
+	basePython := base.With(integration.ProgramTestOptions{
+		Dependencies: []string{
+			filepath.Join("..", "sdk", "python", "bin"),
+		},
+	})
+
+	return basePython
+}
+
+func getCsharpBaseOptions() integration.ProgramTestOptions {
+	base := getBaseOptions()
+	basePython := base.With(integration.ProgramTestOptions{
```

Comment:
csharp ;)

Reply:
I will fix this up - I merged before reading the commend :D